contrib/openvpn/files/openvpn-client

```diff
@@ -0,0 +1,7 @@
+# openvpn client service
+
+type = process
+command = /usr/bin/openvpn --suppress-timestamps --nobind --log /var/log/openvpn-client-service.log --config ${OPENVPN_CLIENT:-default}.conf
```

Comment:
--log erases the file every start which is probably not what you want. there's --log-append instead

but you probably want --syslog openvpn-client instead

Reply:
As somebody who knows nothing and is only asking out of curiosity, when would one prefer a log file vs the system log?

Reply:
we just have no builtin log handling yet so using a log file means it never gets rotated. syslog goes to /var/log/messages and can then be configured to do anything via syslog-ng conf

so generally you'd prefer to send to syslog in this case for now

Reply:
Ah understood thank you for explaining